src/sentry/static/sentry/app/data/forms/projectDebugFiles.jsx

```diff
@@ -75,7 +75,7 @@ export const fields = {
     setValue: raw => (JSON.parse(raw || null) || []).map(flattenKeys),
 
     renderItem(item) {
-      return item.name || <em>{t('<Unnamed Repository>')}</em>;
+      return item.name ? <span>{item.name}</span> : <em>{t('<Unnamed Repository>')}</em>;
```

Comment:
I suppose this is required because of the `display: flex` on the container?

Reply:
Yep.